components/builder-api/habitat/config/config.toml

```diff
@@ -31,3 +31,9 @@ root = "{{pkg.svc_static_path}}"
 
 [upstream]
 {{toToml cfg.upstream}}
+
+{{~#each bind.memcached.members as |member|}}
+[[memcached]]
+host = "{{member.sys.ip}}"
```

Comment:
Am I reading this right, that the `host` portion of your `MemcacheCfg` struct is not used in the config file?